test/service/test_response.py

```diff
@@ -1015,22 +1015,22 @@ def test_filter_with_none(self):
         """
         Test response when using a filter with a None value
         """
-        test_data_values = [["year"], [None], ["year", None]]
+        test_data_values = [["normal"], [None], ["normal", None]]
         for test_data in test_data_values:
             with self.subTest(test_data=test_data):
-                params = self._params(size=10, filters={'organismAgeUnit': {'is': test_data}})
+                params = self._params(size=10, filters={'specimenDisease': {'is': test_data}})
                 url = self.base_url.set(path='/index/samples', args=params)
                 response = requests.get(str(url))
                 response.raise_for_status()
                 response_json = response.json()
-                organism_age_units = {
-                    None if oa is None else oa['unit']
+                diseases = {
+                    disease
                     for hit in response_json['hits']
-                    for donor in hit['donorOrganisms']
-                    for oa in donor['organismAge']
+                    for specimen in hit['specimens']
+                    for disease in specimen['disease']
                 }
                 # Assert that the organismAge unit values found in the response only match what was filtered for
```

Comment:
```suggestion
                # Assert that the values found in the response match only what was filtered for
```

or the comment can be removed.

Reply:
Opted for the removal 👌🏽